docs/shop_administration/manage_products.md

```diff
@@ -51,12 +73,14 @@ Attributes that have this option are used to [automatically generate product var
 
 ## Product availability and stock
 
-To control a product's availability and the available stock, use the product's **Availability** tab.
+To control a product's availability and the available stock, in product or variant view go to **Availability** tab.
 If a product can have variants, you must set availability for each variant separately.
 
 ![Product availability](img/product_availability_tab.png)
 
-Use the switcher to indicate whether the product is currently available in the catalog.
+Click **Create** to add availability to a product that doesn't have it, or **Edit** to modify existing availability settings.
```

Comment:
```suggestion
Click **Create** to add availability to a product that does not have it, or **Edit** to modify existing availability settings.
```

Reply:
There are 2 Edit buttons. I think we should add that the button with edit icon must be clicked to edit the availability.

Reply:
Also added better screenshot which doesn't crop out the button